sources/engine/Stride.BepuPhysics/Stride.BepuPhysics/Constraints/CenterDistanceLimitConstraintComponent.cs

```diff
@@ -9,13 +9,19 @@
 
 namespace Stride.BepuPhysics.Constraints;
 
+/// <summary>
+/// Constrains the center of two bodies to be separated by a distance within a range.
```

Comment:
Maybe mention that `DistanceLimitConstraintComponent` is a variant of this one supporting offset from the center

Reply:
Done.